website/docs/bestpractices/analytics/_category_.json

```diff
@@ -0,0 +1,7 @@
+{
+  "label": "Analytics Best Practices",
```

Comment:
I'd be a bit more specific with `Data Analytics`.

Reply:
Agreed